### DISCUSSION TITLE
Q: Is there an alternative to the "Run with Discrete Graphics" for regolith

### DISCUSSION THREAD

**Discussion by dalyndalton** [regolith-linux/regolith-desktop#995] [General]
laptop user here with hybrid graphics mode enabled.  Is there currently a way to force a program to run with the discrete gpu from the program launcher?

**Comment by kgilmer** [comment]
Hmm, there is no built-in facility for such a feature @dalyndalton .  Do you desire a workflow in which you can select which GPU use upon each launch, or is it more like certain programs have an affinity to a specific GPU?  If it's the latter you might consider creating `desktop` files in your usr dir and configuring the `exec` line to pass whatever GPU flag it might need (assuming that's how GPU affinity is controlled).  FWIW I personally have not had a "good time" w/ hybrid graphics myself w/ Linux.

This might provide some details on user desktop files: https://www.baeldung.com/linux/desktop-entry-files

**Reply by sangemaru** [reply]
Hi @kgilmer , i think @dalyndalton was referring to the option in Ubuntu Gnome to right-click the application and use the "Run with Discrete Graphics" option in order to run an app on the nVidia card using PRIME-offload. This has been working really well in the latest versions of Ubuntu/Nvidia
